hardhat.config.ts

```diff
@@ -1,194 +1,36 @@
-        apiURL: "https://holesky-eth.w3node.com/f1ef94bb8175b1a8f2357a29663a3b8a5b43906d28879e114b8c225a47811c14/api",
-      }
-    },
-    {
-      network: "bitlayer_testnet",
-      chainId: 200810,
-
-      urls: {
-        browserURL: "https://testnet-rpc.bitlayer.org",
-        apiURL: "https://testnet-rpc.bitlayer.org",
-      },
-    },
-    ],
   },
   networks: {
     lorenzo_testnet: {
-      url: "https://rpc-testnet.lorenzo-protocol.xyz",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    movement_testnet: {
-      url: "https://mevm.devnet.imola.movementlabs.xyz/",
+      url: 'https://rpc-testnet.lorenzo-protocol.xyz',
       accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
     },
-    bevm_testnet: {
-      url: "https://testnet.bevm.io",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    linea_sepolia_testnet: {
-      url: "https://linea-sepolia.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    base_sepolia_testnet: {
-      url: "https://sepolia.base.org",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    polygon_amoy: {
-      url: "https://rpc-amoy.polygon.technology",
-      accounts: ["7c5b27c4f043051e405d03469e3f9dfe5b65df74376dcaf70db003d63a976efc"],
-    },
-    citrea_devnet: {
-      url: "https://rpc.devnet.citrea.xyz",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    arbitrum_testnet: {
-      url: "https://arbitrum-sepolia.blockpi.network/v1/rpc/public",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    avalanche_testnet: {
-      url: "https://ava-testnet.public.blastapi.io/ext/bc/C/rpc",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
-    },
-    bob_testnet :{
-      url: "https://testnet.rpc.gobob.xyz",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : []  
-    },
-    holesky_testnet: {
-      url: "https://holesky-eth.w3node.com/f1ef94bb8175b1a8f2357a29663a3b8a5b43906d28879e114b8c225a47811c14/api",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : []
-    },
-    bitlayer_testnet: {
-      url: "https://testnet-rpc.bitlayer.org",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : []
-    },
-    core_testnet: {
-      url: "https://rpc.test.btcs.network",
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : []
+    scroll_testnet: {
+      url:'https://scroll-sepolia.drpc.org',
+      chainId:534351,
+      accounts: [
+        'a0b8d309dd0afc95c94fc319f2de434f845a747d662c71ccbc08655c29299519',
+      ],
     },
   },
   solidity: {
     compilers: [
       {
-        version: "0.8.19",
+        version: '0.8.19',
```

Comment:
Your formatting is still different from the `.prettierc` that is configured in the code. Try running `npm run format`. And also check the `.prettierc` file